components/Favorites/FavoriteItemsBar.tsx

```diff
@@ -0,0 +1,197 @@
+                                    <Image src={iconSrc} alt="" width={40} height={40} />
+                                </div>
+                                <div className={styles.details}>
+                                    <span className={styles.name}>{itemName}</span>
+                                    <div className={styles.stats}>
+                                        <span
+                                            className={`${styles.delta} ${delta !== null ? (isPositive ? styles.positive : styles.negative) : ''} ${
+                                                isMovementLoading ? styles.skeletonInline : ''
+                                            }`}
+                                            title={monthlyTooltip}
+                                        >
+                                            {movement && delta !== null
+                                                ? delta === 0
+                                                    ? 'No change'
+                                                    : `${isPositive ? '+' : ''}${numberWithThousandsSeparators(Math.round(delta))} coins`
+                                                : isMovementLoading
+                                                  ? ''
+                                                  : 'No data'}
+                                            {movement && percentChange !== null && delta !== 0 ? (
+                                                <span className={styles.percent}>
+                                                    {' '}
+                                                    ({`${percentChange >= 0 ? '+' : ''}${percentChange.toFixed(1)}%`})
+                                                </span>
+                                            ) : null}
+                                        </span>
+                                        {movement && currentPrice !== undefined ? (
+                                            <Badge bg="secondary">Now: {numberWithThousandsSeparators(Math.round(currentPrice))}</Badge>
+                                        ) : isMovementLoading ? (
+                                            <span className={`${styles.skeletonInline} ${styles.badgeSkeleton}`} aria-hidden="true"></span>
+                                        ) : null}
+                                    </div>
+                                </div>
+                            </Link>
+                            <div className={styles.actions}>
+                                <SubscribeButton
+                                    topic={favorite.tag}
+                                    type={favorite.bazaar ? 'bazaar' : 'item'}
+                                    buttonContent={<span>Notify</span>}
+                                />
+                                <FavoriteToggle item={item} size="small" />
+                            </div>
+                        </article>
+                    )
```

Comment:
This should probably be a own component but that is only my preference

Reply:
refactored